### PR TITLE
Clarify Nextflow Descriptor for GitHub Apps

### PR DESCRIPTION
**Pull request by nolwarre**
Ticket: https://ucsc-cgl.atlassian.net/browse/SEAB-1802

Differed from the examples and was not clear on which file to use for Nextflow descriptor.

### CODE REVIEW

File: docs/getting-started/github-apps/github-apps.rst [hunk 1]
```diff
@@ -82,7 +82,7 @@ name (optional)
 subclass
     The descriptor language used for the workflow. Supported values include CWL, WDL, NFL (Nextflow), and GXFORMAT2 (Galaxy). This cannot be changed once the workflow is registered.
 primaryDescriptorPath
-    The absolute path to the primary descriptor file in the Git repository.
+    The absolute path to the primary descriptor file in the Git repository. For Nextflow, the primary descriptor is the .config file.
```
Comment (coverbeck) [inline docs/getting-started/github-apps/github-apps.rst:85]: Isn't it typically `nextflow.config`? If so, we should say that.

This feels a little out of place to me. I feel like like we should mention all languages or none. Maybe have sub-bullets? For Galaxy, the primary descriptor is a .ga file; for WDL the primary descriptor file is a .wdl file, etc.

But if I'm the only who feels this way, I'm OK with.

**Review by coverbeck — COMMENTED** [review]
Not part of your changes, but because I happened to notice it, can you please change `GXFORMAT2` to `GALAXY` on line 70?

**Review by wshands — APPROVED** [review]

File: docs/getting-started/github-apps/github-apps.rst (hunk 1 above)
Comment (wshands) [reply docs/getting-started/github-apps/github-apps.rst]: 👍

**Comment by wshands** [comment]
Yes please fix line 70 as Charles mentioned

**Review by denis-yuen — APPROVED** [review]

**Review by coverbeck — APPROVED** [review]

**Pull request merged**